test/tests/functional/pbs_soft_walltime.py

```diff
@@ -889,8 +889,8 @@ def test_soft_held(self):
             " since the walltime was reset to 0")
         time.sleep(17)
         self.server.expect(JOB, {'estimated.soft_walltime':
-                                 (MATCH_RE, '00:00:21|21')}, id=jid,
-                           max_attempts=5)
+                                 (MATCH_RE, '00:00:21|21|00:00:28|28')},
+                           id=jid)
```

Comment:
I've recently learned that it's the server that converts a time based resource into HH:MM:SS.  You don't actually have to do the MATCH_RE.  Regardless of cli/api mode, it the resource will be in that format.  There's no harm to what you have done.  It is consistent with the rest of the test.

Reply:
Thanks for the sharing Bhroam. Like you said I would leave it like that to be consistent with rest of the test.